aiohttp/streams.py

```diff
@@ -604,6 +605,18 @@ def read_nowait(self, n: int = -1) -> bytes:
 EMPTY_PAYLOAD: Final[StreamReader] = EmptyStreamReader()
```

Comment:
We'll probably want a followup PR to remove EMPTY_PAYLOAD too, without backporting.